Formula/r.rb

```diff
@@ -127,6 +123,13 @@ def install
       s.gsub!(/.LDFLAGS =.*/, "\\0 $(LDFLAGS)")
     end
 
+    # Avoid implicit declarations producing compile-time errors, as seen in
+    # Xcode 12+. This used to be necessary to build R itself, but now should
+    # only be necessary for packages.
+    inreplace r_home/"etc/Makeconf" do |s|
```

Comment:
Do we need a new `inreplace` block?
```suggestion
      # Avoid implicit declarations producing compile-time errors, as seen in
      # Xcode 12+. This used to be necessary to build R itself, but now should
      # only be necessary for packages.
```

Reply:
Thanks for the review. I separated them because I thought it made their respective (distinct) intentions clearer, but of course they can be combined if preferred. The change is incorporated.